module/zstd/zstd.c

```diff
@@ -622,7 +622,7 @@ zstd_mempool_deinit(void)
 	    ZSTD_POOL_MAX * sizeof (struct zstd_pool));
 }
 
-extern int __init
+static int __init
```

Comment:
don;t forget to reset this change when reedding the _init elsewhere

Reply:
yes. i made here some specific changes to treat it different in userspace to export it. in kernel mode it can be static and it does need the attribute __init. but the static is not required. important is __init and __exit. so if _KERNEL is defined. a simple #define __init helps here. just like i did